packages/app/src/runner/selector-playground/SelectorPlayground.vue

```diff
@@ -1,83 +1,135 @@
 <template>
   <div
     id="selector-playground"
-    class="bg-white flex items-center"
+    class="border-t border-b bg-gray-50 border-gray-200 h-56px grid p-12px gap-12px grid-cols-[40px,1fr,auto] items-center "
   >
     <button
-      :class="{ 'bg-blue-100': selectorPlaygroundStore.isEnabled }"
-      class="rounded-md h-full px-8px"
+      class="border rounded-md flex h-full outline-none text-white transition w-40px duration-150 items-center justify-center"
+      :class="{'default-ring': selectorPlaygroundStore.isEnabled, 'border-gray-200': !selectorPlaygroundStore.isEnabled}"
       data-cy="playground-toggle"
       @click="toggleEnabled"
     >
-      <Icon
-        :icon="IconCursorDefaultOutline"
-        height="18px"
-        width="18px"
-      />
+      <i-cy-selector_x16 :class="{ 'icon-dark-indigo-500': selectorPlaygroundStore.isEnabled, 'icon-dark-gray-500': !selectorPlaygroundStore.isEnabled }" />
     </button>
-
     <div
-      class="flex h-full flex-1 mx-2 items-center"
+      class="flex h-full flex-1 w-full relative items-center"
       @mouseover="setShowingHighlight"
     >
-      <Select
-        :options="methods"
-        item-value="display"
-        data-cy="playground-method"
-        :model-value="{
-          value: selectorPlaygroundStore.method,
-          display: `cy.${selectorPlaygroundStore.method}`
-        }"
-        class="w-120px"
-        @update:model-value="({ value }) => selectorPlaygroundStore.setMethod(value)"
-      />
-
-      <span class="mx-5px">('</span>
-      <input
-        ref="copyText"
-        v-model="selector"
-        class="border rounded-md border-gray-500 flex-1 py-8px px-1 pl-4 text-blue-500"
-        data-cy="playground-selector"
-      >
-      ')
-    </div>
-
-    <div
-      data-cy="playground-num-elements"
-      class="rounded-md flex h-full bg-gray-400 mx-1 text-white px-3 items-center"
-    >
-      {{ selectorPlaygroundStore.numElements }}
+      <Menu #="{ open }">
+        <MenuButton
+          aria-label="Playground Methods"
```

Comment:
Maybe something like `Selector Options` would be more descrpitive? I'm not sold on it either but I don't know if Playground Methods would convey what these are

Reply:
Selector options makes more sense